app/views/assets/samples_partials/_tube_rack_samples.erb

```diff
@@ -0,0 +1,48 @@
+<section id="samples-table" class='info-panel'>
+  <header>
+    <a href='#plate-samples-table-collapser' role="button" data-toggle='collapse'>
+    <h3>Samples
+      <%= counter_badge @asset.contained_samples.count %>
+      <small class='show-hide'>Click to toggle</small>
+    </h3>
+    </a>
+  </header>
+  <div id="plate-samples-table-collapser" class="collapse in show">
```

Comment:
What is this id used for? Is it possible to change it to not use the word 'plate'?

Reply:
(and the line below)

Reply:
Yes, I'm changing ids to tube rack.  

It is use for relating the collapsed tab with the tab content and collapse and uncollapse in Javascript